lib/chibios-syscalls/dynamic_alloc_stubs.cpp

```diff
@@ -0,0 +1,33 @@
+#include <ch.h>
+#include <hal.h>
+
+#include <stdio.h>
+#include <errno.h>
+
+#ifndef CVRA_NO_DYNAMIC_ALLOCATION
+# define CVRA_NO_DYNAMIC_ALLOCATION 0
+#endif
+
+#if CVRA_NO_DYNAMIC_ALLOCATION
```

Comment:
The default implementation of `operator new()` can throw a `bad_alloc` exception, which we don't want.
I think until now we never had code that called `new()`, so it's not a problem at the moment to leave the default implementation.
So, just to keep in mind...